api/extensions/ext_mail.py

```diff
@@ -84,7 +86,11 @@ def send(self, to: str, subject: str, html: str, from_: Optional[str] = None):
         )
 
 
-def init_app(app: Flask):
+def is_enabled() -> bool:
+    return dify_config.MAIL_TYPE is not None
```

Comment:
This is not skipping the ext_mail when `MAIL_TYPE` is set with an empty string , e.g. `MAIL_TYPE=` in `.env.example`.

Reply:
Thanks for the notice, I will fix it.

Reply:
Or let's change it into `return dify_config.MAIL_TYPE is not None and dify_config.MAIL_TYPE.strip() != ""`

Reply:
In fact, I believe it will be a configuration error if this value is blank. We do not need to care about it.

Reply:
In fact, I believe it will be a configuration error if this value is blank. We do not need to care about it.